src/Flight/FlightService.java

```diff
@@ -1,6 +1,143 @@
 package Flight;
 
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+
 // TODO: Implement the Service Layer, this contains all Business Logic (anything that makes sense in the scope of a business)
 public class FlightService {
+    private Flight[] flights = new Flight[5];
+
+
+    public Flight[] getFlightInfo(){
+        if (isEmpty(flights)) {
+            System.out.println("No flight info");
+            return null;
+        } else {
+            System.out.println("Returning flight information....");
+            return flights;
+        }
+    }
+
+    public void createFlight(Flight flight){
+        int indexToReplace = getFirstNull(flights);
+        if (indexToReplace == -1) {
+            System.out.println("Sorry, our flight database is full, please try again later");
```

Comment:
Will there be a point where we will use a list or a dynamic array to ensure there can be more than 5 flights at a time?